File: Modul-4/Soal-3/PRAK403-2410817310012-FAISALTANJUNG.py
nilai1, nilai2 = map(int, input().split())

i = nilai1
j = nilai2

if i < j:
    while i<=nilai2 and j>=nilai1:
        print(f"{i} {j}", end=" ") 
        if i < nilai2 or j > nilai1:
            print("-", end=" ")
        i +=1
        j -=1   
else:
    while i >= nilai2 and j <= nilai1:
        print(f"{i} {j}", end=" ")
        if i > nilai2 or j < nilai1:
            print("-", end=" ")
        i -=1
        j +=1

File: Modul-4/Soal-3/PRAK403-2410817310012-FAISALTANJUNG.c
#include <stdio.h>
int main (){
    int nilai1, nilai2;

    printf("");
    scanf("%d %d", &nilai1, &nilai2);

    for (int i=nilai1, j=nilai2; i<=nilai2 && j>=nilai1; i++, j--) {
        printf("%d %d", i, j);
        if (i<nilai2 || j>nilai1) {
            printf(" - ");
        }
    }

    for (int i=nilai1, j= nilai2; i >= nilai2 && j <= nilai1; i--, j++) {
        printf("%d %d", i, j);
        if (i > nilai2 || j < nilai1) {
            printf(" - ");
        }
    }
    printf("\n\n");
    return 0;
} 